metagraph/src/annotation/int_matrix/extended/csc_matrix.hpp

```diff
@@ -57,62 +57,62 @@ class ColumnRankMatrix : public CountMatrix {
 
   private:
     BaseMatrix binary_matrix_;
-    std::vector<sdsl::int_vector<>> count_columns_;
+    std::vector<sdsl::int_vector<>> column_values_;
 };
 
 
 template <class BaseMatrix>
-inline typename ColumnRankMatrix<BaseMatrix>::ColumnCounts
-ColumnRankMatrix<BaseMatrix>::get_row_counts(Row row) const {
-    const auto &column_ranks = binary_matrix_.get_row_ranks(row);
-    ColumnCounts row_counts;
-    row_counts.reserve(column_ranks.size());
+inline typename CSCMatrix<BaseMatrix>::RowValues
+CSCMatrix<BaseMatrix>::get_row_values(Row row) const {
+    const auto &column_ranks = binary_matrix_.get_column_ranks(row);
+    RowValues row_values;
+    row_values.reserve(column_ranks.size());
     for (auto [j, r] : column_ranks) {
         assert(r >= 1 && "matches can't have zero-rank");
-        row_counts.emplace_back(j, count_columns_[j][r - 1]);
+        row_values.emplace_back(j, column_values_[j][r - 1]);
     }
-    return row_counts;
+    return row_values;
 }
 
 template <class BaseMatrix>
-inline std::vector<typename ColumnRankMatrix<BaseMatrix>::ColumnCounts>
-ColumnRankMatrix<BaseMatrix>::get_row_counts(const std::vector<Row> &rows) const {
-    const auto &column_ranks = binary_matrix_.get_row_ranks(rows);
-    std::vector<ColumnCounts> row_counts(rows.size());
+inline std::vector<typename CSCMatrix<BaseMatrix>::RowValues>
+CSCMatrix<BaseMatrix>::get_row_values(const std::vector<Row> &rows) const {
+    const auto &column_ranks = binary_matrix_.get_column_ranks(rows);
```

Comment:
Same comment as above
```suggestion
    auto column_ranks = binary_matrix_.get_column_ranks(rows);
```